### PR TITLE
Clean Ups

### PR DESCRIPTION
**Pull request by Lord-Haji**

### CODE REVIEW

**Comment by Lord-Haji** [comment]
#22

File: README.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ Navigation: [Spacialgaze][1] | [PS Server repository][10] | [PS Client repositor
   [10]: https://github.com/Zarel/Pokemon-Showdown
 
 [![Build Status](https://travis-ci.org/HoeenCoder/SpacialGaze.svg)](https://travis-ci.org/HoeenCoder/SpacialGaze)
+[![dependencies Status](https://david-dm.org/HoeenCoder/SpacialGaze/status.svg)](https://david-dm.org/HoeenCoder/SpacialGaze)
+[![devDependencies Status](https://david-dm.org/HoeenCoder/SpacialGaze/dev-status.svg)](https://david-dm.org/HoeenCoder/SpacialGaze?type=dev)
+[![optionalDependencies Status](https://david-dm.org/HoeenCoder/SpacialGaze/optional-status.svg)](https://david-dm.org/HoeenCoder/SpacialGaze?type=optional)
```
Comment (HoeenCoder) [inline README.md:14]: None of these will work unless I set them up, I might do so in the future but today you should just remove them.

File: package.json [hunk 2]
```diff
@@ -5,19 +5,19 @@
...
+    "request": "2.79.0",
+    "md5": "2.2.1",
+    "moment": "2.17.1",
+    "geoip-ultralight": "0.1.4"
   },
   "optionalDependencies": {
-    "cloud-env": "0.1.1",
-    "http-proxy": "0.10.0",
-    "nodemailer": "1.4.0",
-    "node-static": "0.7.7"
```
Comment (HoeenCoder) [inline package.json:20]: Please dont edit these, let main update them when ready.

File: package.json [hunk 3]
```diff
@@ -5,19 +5,19 @@
...
   "dependencies": {
     "sockjs": "0.3.18",
-    "sqlite3": "*",
-    "origindb": "*",
-    "autolinker": "*",
-    "request": "*",
-    "md5": "*",
-    "moment": "*",
-    "geoip-ultralight": "*"
+    "sqlite3": "3.1.8",
```
Comment (HoeenCoder) [inline package.json]: `*` already refers to the latest version.

**Review by HoeenCoder — COMMENTED** [review]
Looks good, just a few minor changes needed first.

File: README.md (hunk 1 above)
Comment (HoeenCoder) [reply README.md:14]: Nevermind this will work

**Comment by HoeenCoder** [comment]
Ill work on the sqlite3 later, we should probably just depreciate and refactor to using `Db` but I want to talk to @Mystifi first.

**Pull request merged**